packages/caliper-vscode/extension/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This should extend the available Node14 base:
* How-to: https://www.typescriptlang.org/docs/handbook/tsconfig-json.html#tsconfig-bases
* Needed package: @tsconfig/node14
* Content of base file: https://github.com/tsconfig/bases/blob/main/bases/node14.json